snowshu/core/graph_set_runner.py

```diff
@@ -173,8 +174,9 @@ def _traverse_and_execute(self, executable: GraphExecutable) -> None:  # noqa: p
                 f"Executing graph with {len(executable.graph)} relations in it...")
             sorted_graphs = nx.algorithms.dag.topological_sort(executable.graph)
             for i, relation in enumerate(sorted_graphs, start=1):
-
-                relation.temp_schema = "_".join([relation.schema, self.uuid])
+                relation.temp_schema = "_".join(
```

Comment:
Here's major change, we needed to make schemas unique, due to the fact that we encountered a scenario where we process realtions that comes from different databases, but the same schemas and table names causing naming collisions.